virt-v2v/cold/entrypoint.go

```diff
@@ -132,6 +132,17 @@ func customizeVM(source string, xmlFilePath string) error {
 				return err
 			}
 		}
+
+		// RHEL
+		if strings.Contains(operatingSystem, "rhel") {
+			t := EmbedTool{filesystem: &scriptFS}
+
+			err = CustomizeRHEL(CustomizeDomainExec, disks, DIR, &t)
```

Comment:
for testing, in the testing i mock CustomizeDomainExec, because I don't want to actually run the virt-customize command, the mock method just printout the args